polyfills/Element/prototype/getAttributeNames/config.json

```diff
@@ -0,0 +1,31 @@
+{
+    "aliases": [
+        "default-3.3",
```

Comment:
Right now I don't think we want to add this polyfill to any of the default sets.

In the future we may add it into the default set 👍